frontend/src/metabase/admin/settings/app/components/SettingsEditor/tests/allowed-domains.unit.spec.tsx

```diff
@@ -0,0 +1,32 @@
+import { screen } from "__support__/ui";
+import {
+  createMockSettingDefinition,
+  createMockSettings,
+  createMockTokenFeatures,
+} from "metabase-types/api/mocks";
+
+import { EMAIL_URL, setup } from "./setup";
+
+async function setupAllowedDomains() {
+  await setup({
```

Comment:
I made `setup` async and updated all the tests to `await` it properly, see my comment in `setup.tsx`